src/GCHP_GridComp/GCHPctmEnv_GridComp/GCHPctmEnv_GridCompMod.F90

```diff
@@ -708,6 +720,23 @@ subroutine Run ( GC, IMPORT, EXPORT, CLOCK, RC )
       !DEALLOCATE( UCr8, VCr8, PLEr8, PLE0, PLE1, DryPLE0, DryPLE1 )
       DEALLOCATE( UCr8, VCr8, PLEr8, UC, VC)
 
+      ! Vertical motion diagnostics
+      call MAPL_GetPointer ( EXPORT, UpwardsMassFlux, 'UpwardsMassFlux', ALLOC=.TRUE., RC=STATUS )
+      _VERIFY(STATUS)
+
+      ! Get vertical mass flux
+      call fv_getVerticalMassFlux(MFXr8, MFYr8, UpwardsMassFlux, dt)
+
+      ! Flip vertical so that GCHP diagnostic is positive="up"
+      UpwardsMassFlux(:,:,:) = UpwardsMassFlux(:,:,LM:0:-1)
```

Comment:
In newer MAPL I believe we will be able to specify in HISTORY.rc whether to flip to positive up.  When that comes in all MAPL exports would be stored as positive down so this flipping would need to be removed.